api-tests/management/dbaas/kubernetes_server_test.go

```diff
@@ -23,8 +23,9 @@ import (
 	"time"
 
 	dbaasClient "github.com/percona/pmm/api/managementpb/dbaas/json/client"
+	dbclusters "github.com/percona/pmm/api/managementpb/dbaas/json/client/db_clusters"
 	"github.com/percona/pmm/api/managementpb/dbaas/json/client/kubernetes"
-	psmdbcluster "github.com/percona/pmm/api/managementpb/dbaas/json/client/psmdb_cluster"
+	psmdbclusters "github.com/percona/pmm/api/managementpb/dbaas/json/client/psmdb_clusters"
```

Comment:
is it related to the TT ?

Reply:
No, in both cases. I am using PMM-9063-okta as target branch, so that is why. Will check target branch.

Reply:
@maxkondr It should be fixed now, I merged 9063 into this branch.